concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -0,0 +1,345 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	Sound_symbolic
```

Comment:
We have the convention to write uppercase throughout concepticon. While we could break this in the future, I'd then also require all otehr cols to be reviewed, and since this is a lot of work, I suggest to write SOUND_SYMBOLIC